hydroshare/urls.py

```diff
@@ -58,9 +58,6 @@
     url(r'^delete_irods_account/$', theme.delete_irods_account, name='delete_irods_account'),
     url(r'^create_irods_account/$', theme.create_irods_account, name='create_irods_account'),
     url(r'^accounts/login/$', theme.login, name='login'),
```

Comment:
should sync up with the latest updates to dashboard code and re-test

Reply:
Synced with current develop.